cfgov/unprocessed/apps/tccp/css/main.less

```diff
@@ -7,89 +7,79 @@
-  }
-
-  input[type='checkbox'],
-  input[type='radio'] {
-    margin-right: 4px;
-  }
-}
-
-.m-btn-group {
-  .u-btn-helper {
-    width: 100%;
-    margin-top: unit((15px / @base-font-size-px), rem);
-    text-align: center;
-    a {
-      border-bottom-width: 1px;
+// A form fieldset with large block-level areas for
+// helper text under each form field
+.o-form__fieldset--helpers {
+  .a-checkbox:checked + .a-label {
+    border: 1px solid var(--pacific);
+    border-bottom: 0;
+    box-shadow: none;
+  }
+
+  .m-form-field--checkbox {
+    &:has(.a-checkbox:focus) {
+      outline: 1px dotted var(--pacific);
+      outline-offset: 1px;
     }
-    .respond-to-min(@bp-sm-min, {
-      display: inline-block;
-      width: auto;
-      margin-top: 0;
-      margin-left: unit((6px / @btn-font-size), rem);
-    });
-  }
-}
-
-.a-checkbox + .a-label > .a-label__helper {
-  display: none;
-}
 
-.a-checkbox:checked + .a-label > .a-label__helper {
-  display: grid;
-  grid-column-start: 1;
-  grid-column-end: 3;
-  margin-top: 1em;
-  margin-bottom: 1em;
-}
+    .a-checkbox:focus + .a-label {
+      outline: none;
+    }
 
-.a-label__helper .m-list {
-  list-style: square;
-  padding-left: 1.125em;
+    .a-checkbox ~ [data-helper-text] {
+      display: none;
+      box-sizing: border-box;
+      width: 100%;
+      padding: unit((15px / @base-font-size-px), rem);
+      padding-top: 0;
+      background-color: var(--pacific-20);
+      border: 1px solid var(--pacific);
+      border-top: 0;
+    }
 
-  .m-list__item {
-    margin-left: unit((15px / @base-font-size-px), rem);
+    .a-checkbox:checked ~ [data-helper-text] {
+      display: block;
+    }
   }
 
-  .m-list__item--has-icon {
-    display: flex;
-    margin-left: 0;
+  [data-helper-text] > .m-list {
+    list-style: square;
+    padding-left: 1.125em;
 
-    .cf-icon-svg {
-      display: inline;
-      margin-right: 6px;
-      color: var(--green);
+    .m-list__item {
+      margin-left: unit((15px / @base-font-size-px), rem);
     }
 
-    span {
-      margin-top: unit((-1px / @base-font-size-px), rem);
-    }
-  }
-}
+    .m-list__item--has-icon {
+      display: flex;
+      margin-left: 0;
 
-.tool-features {
-  .cf-icon-svg--credit-card-round {
-    color: var(--gold);
-  }
-
-  .cf-icon-svg--court-round {
-    color: var(--gray);
-  }
+      .cf-icon-svg {
+        display: inline;
+        margin-right: 6px;
+        color: var(--green);
+      }
 
-  .cf-icon-svg--disabled-round {
-    color: var(--red);
+      span {
+        margin-top: unit((-1px / @base-font-size-px), rem);
+      }
+    }
   }
+}
 
-  .cf-icon-svg--dollar-round {
-    color: var(--green);
+.u-btn-helper {
+  width: 100%;
+  margin-top: unit((15px / @base-font-size-px), rem);
+  text-align: center;
+  a {
+    border-bottom-width: 1px;
   }
+  .respond-to-min(@bp-sm-min, {
+    display: inline-block;
+    width: auto;
+    margin-top: 0;
+    margin-left: unit((6px / @btn-font-size), rem);
```

Comment:
Doesn't matter for now because we have a mixed style across the codebase, but I have begun removing the nested parentheses in the `unit(…)`. We used to need this for the linter I think, but no longer.